sdk/core/core-amqp/src/retry.ts

```diff
@@ -66,14 +71,29 @@ export interface RetryConfig<T> {
   maxRetries?: number;
   /**
    * @property {number} [delayInSeconds] Amount of time to wait in seconds before making the
-   * next attempt. Default: 15.
+   * next attempt. Applicable only when performing linear retry. Default: 15.
```

Comment:
Can we not use this instead of introducing `minExponentialRetryDelayInMs`?

Reply:
What would be the benefits?
One is intended for with linear retry where min/max don't lend any meaning.
For exponential retry, we have notion of min and max, so that would likely be a con as it may be confusing to mix these into one. Unless maybe the naming is clear enough?
@bterlson thoughts on usage/name?

Reply:
Ah, never mind. I was looking at the storage implementation where there is no "min" value and the  delay for the first time is the same as `delayInSeconds`